src/local_scheduler/local_scheduler_client.cc

```diff
@@ -15,8 +15,10 @@ using MessageType = ray::local_scheduler::protocol::MessageType;
 LocalSchedulerConnection *LocalSchedulerConnection_init(
     const char *local_scheduler_socket,
     UniqueID client_id,
-    bool is_worker) {
+    bool is_worker,
+    bool use_raylet) {
```

Comment:
@salah-man, since I changed the C++ API for `LocalSchedulerConnection_init`, I had to change a few Java files. Can you take a look that the change there is ok?

Reply:
Yes, i just find that there is only one change in org_ray_spi_impl_DefaultLocalSchedulerClient.cc for the upgrade of LocalSchedulerConnection_init, and i think it looks good to me.